lib/vtls/schannel_verify.c

```diff
@@ -630,31 +631,46 @@ CURLcode Curl_verify_certificate(struct Curl_cfilter *cf,
       result = CURLE_SSL_CACERT_BADFILE;
     }
     else {
-      /* Open the certificate store */
-      trust_store = CertOpenStore(CERT_STORE_PROV_MEMORY,
-                                  0,
-                                  (HCRYPTPROV)NULL,
-                                  CERT_STORE_CREATE_NEW_FLAG,
-                                  NULL);
-      if(!trust_store) {
-        char buffer[STRERROR_LEN];
-        failf(data, "schannel: failed to create certificate store: %s",
-              Curl_winapi_strerror(GetLastError(), buffer, sizeof(buffer)));
-        result = CURLE_SSL_CACERT_BADFILE;
+      /* try cache */
+      trust_store = Curl_schannel_get_cached_cert_store(cf, data);
```

Comment:
openssl get/set of cert store has more conditionals, see `cache_criteria_met`. it looks like most? of those don't apply here but what about data->set.general_ssl.ca_cache_timeout

https://github.com/curl/curl/blob/3e6254f819070eb82c72e94c4469d9baaddfe2f4/lib/vtls/openssl.c#L3439-L3460

Reply:
Thanks. Zero cache timeout should skip cache usage at all. Will fix with next commit.